ipfscluster_test.go

```diff
@@ -1897,3 +1898,84 @@ func TestClustersDisabledRepinning(t *testing.T) {
 		t.Errorf("expected %d replicas for pin, got %d", nClusters-2, numPinned)
 	}
 }
+
+func TestClustersFollowerMode(t *testing.T) {
+	ctx := context.Background()
+	clusters, mock := createClusters(t)
+	defer shutdownClusters(t, clusters, mock)
+
+	_, err := clusters[0].Pin(ctx, test.Cid1, api.PinOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = clusters[0].Pin(ctx, test.ErrorCid, api.PinOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
don't really need above two tests.

Reply:
yes, the above is needed so the `sync` test below works.